### PR TITLE
add all functional

### PR DESCRIPTION
**Pull request by Illia1927**

### CODE REVIEW

File: src/main/java/mate/academy/spring/config/AppInit.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import org.springframework.web.context.support.AnnotationConfigWebApplicationContext;
+import org.springframework.web.servlet.DispatcherServlet;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletRegistration;
+
+public class AppInit implements WebApplicationInitializer {
+
+    public void onStartup(ServletContext container) {
+
```
Comment (DmitriyLitvin) [inline src/main/java/mate/academy/spring/config/AppInit.java:13]: empty line

File: src/main/java/mate/academy/spring/config/DataConfig.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import javax.sql.DataSource;
+import java.util.Properties;
+
+@Configuration
+@EnableTransactionManagement
+@ComponentScan(value = "mate.academy.spring")
+@PropertySource("classpath:application.properties")
+@EnableJpaRepositories("mate.academy.spring.repository")
+public class DataConfig {
+
```
Comment (DmitriyLitvin) [inline src/main/java/mate/academy/spring/config/DataConfig.java:29]: empty line

File: src/main/java/mate/academy/spring/controller/AuthController.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+public class AuthController {
+
```
Comment (DmitriyLitvin) [inline src/main/java/mate/academy/spring/controller/AuthController.java:19]: empty line

File: src/main/java/mate/academy/spring/controller/DeveloperController.java [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.annotation.PostConstruct;
+import java.util.List;
+
+@Log4j2
+@Controller
+@RequestMapping("/developer")
+public class DeveloperController {
+
```
Comment (DmitriyLitvin) [inline src/main/java/mate/academy/spring/controller/DeveloperController.java:22]: empty line

File: src/main/java/mate/academy/spring/controller/DeveloperController.java [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+        model.put("developers", developers);
+        return "developer/listDeveloper";
+    }
+
+    @PostConstruct
+    public void postConstruct() {
+        log.info("Developer controller OK!");
+    }
+}
```
Comment (DmitriyLitvin) [inline src/main/java/mate/academy/spring/controller/DeveloperController.java:102]: a lot of empty lines in methods, remove them, and you don't need log.info("controller: DEVELOPER LIST") here

File: src/main/java/mate/academy/spring/model/Skill.java [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import java.util.Set;
```
Comment (DmitriyLitvin) [inline src/main/java/mate/academy/spring/model/Skill.java:15]: javax and java are differrent imports, so they must be separated by empty line

File: src/main/java/mate/academy/spring/service/impl/DeveloperServiceImpl.java [hunk 7]
```diff
@@ -0,0 +1,58 @@
+package mate.academy.spring.service.impl;
+
+import lombok.extern.log4j.Log4j;
+import mate.academy.spring.dto.DeveloperDto;
+import mate.academy.spring.model.Developer;
+import mate.academy.spring.repository.DeveloperRepository;
+import mate.academy.spring.service.DeveloperService;
+import org.springframework.beans.factory.annotation.Autowired;
```
Comment (DmitriyLitvin) [inline src/main/java/mate/academy/spring/service/impl/DeveloperServiceImpl.java:9]: the same situation with java and javax you have here, so chack it everywere

File: src/main/java/mate/academy/spring/service/impl/DeveloperServiceImpl.java [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+                .stream()
+                .map(DeveloperDto::doDtoDev)
+                .collect(Collectors.toList());
+    }
+
+    @Override
+    public void delete(Long id) {
+        developerRepository.deleteById(id);
+    }
+}
```
Comment (DmitriyLitvin) [inline src/main/java/mate/academy/spring/service/impl/DeveloperServiceImpl.java:59]: @Illia1927 Your main problem is empty lines, remove them, I don't want to write it everywere

File: src/main/webapp/WEB-INF/pages/developer/createDeveloper.jsp [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+    <input  name="developerAge" type="text" id="developerAge" placeholder="developerAge">
+
+    <span class="fontawesome-user"></span>
+    <input  name="developerSalary" type="text" id="developerSalary" placeholder="developerSalary">
+
+    <input type="submit" value="Submit"/>
+    </form>
+</body>
+    </div>
+</html>
```
Comment (DmitriyLitvin) [inline src/main/webapp/WEB-INF/pages/developer/createDeveloper.jsp]: end of file, you have to add empty line and do it everywere

**Review by DmitriyLitvin — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/spring/controller/DeveloperController.java (hunk 5 above)
Comment (Illia1927) [reply src/main/java/mate/academy/spring/controller/DeveloperController.java:102]: @DmitriyLitvin Why? I want to see that this method are working...

File: src/main/java/mate/academy/spring/model/Skill.java (hunk 6 above)
Comment (Illia1927) [reply src/main/java/mate/academy/spring/model/Skill.java:15]: @DmitriyLitvin  Ok

**Pull request merged**